metadata/src/test/java/org/apache/kafka/controller/QuorumControllerTest.java

```diff
@@ -173,6 +176,121 @@ private void testDelayedConfigurationOperations(LocalLogManagerTestEnv logEnv,
         assertEquals(Collections.singletonMap(BROKER0, ApiError.NONE), future1.get());
     }
 
+    @Test
+    public void testFenceMultipleBrokers() throws Throwable {
+        int brokerCount = 5;
+        int brokersToKeepUnfenced = 1;
+        short replicationFactor = 5;
+        Long sessionTimeoutSec = 1L;
```

Comment:
nit: maybe we can use milliseconds consistently and leave out some unit conversions below.

Also, could this and `sleepMillis` be a `long` instead of `Long`?

Reply:
Will make the change. The issue is that the sleepTimeout configuration takes in seconds and I wanted to skip the coversion there. But I will make the time unit consistent across the test.